tests/test_inner_functions.py

```diff
@@ -200,6 +200,17 @@ def test_set_comprehension_func(value):
     assert win.call()
 
 
+# Test that we can access the non local values and the closure variables
+@pytest.mark.parametrize('value', ['self', 'test'])
+def test_generator_expression_func(value):
+    """Test running a generator expression in a declarative function.
+
+    """
+    source = FUNCTION_TEMPLATE.format('any(i for i in range(10))')
```

Comment:
You need to use value like in other tests otherwise you do not test access to non local values and closures.